packages/orbit-components/.storybook/orbitDecorator.tsx

```diff
@@ -16,6 +19,16 @@ const OrbitDecorator = (storyFn, context) => {
     };
   }, []);
 
+  const handleCopyCode = codeToCopy => {
+    navigator.clipboard.writeText(codeToCopy);
```

Comment:
We have a type warning here that we can avoid if we define `codeToCopy` to be of type `string`